python/lsst/ctrl/bps/wms_service.py

```diff
@@ -153,9 +153,8 @@ class WmsRunReport:
     job_summary: dict[str, dict[WmsStates, int]] = None
     """Job counts per label and per state."""
 
-    exit_code_summary: dict[list] = None
-    """Summary of non-zero exit codes per job label
-    available through the WMS.
+    exit_code_summary: dict[str, list[int]] = None
+    """Summary of exit codes per job label available through the WMS.
```

Comment:
Should clarify what this means for jobs other than finished state (e.g., cancelled, running, pending, held due to an exit code, exit by signal, etc).   I don't have strong opinions on which way it should be done, but do prefer that the definition is such that the list contents are mostly the same across WMS.